jdbcSource/src/test/java/io/vantiq/extsrc/jdbcSource/TestJDBC.java

```diff
@@ -32,6 +32,7 @@
 import io.vantiq.client.VantiqResponse;
 import io.vantiq.extsrc.jdbcSource.exception.VantiqSQLException;
 
+@SuppressWarnings("PMD.ExcessiveClassLength")
```

Comment:
this is how you suppress sider warnings about things we don't care about here.  In this case, this may be a good marker to revisit things, but I'm not taking that on right now.